.env

```
@@ -11,4 +11,4 @@ POSTGRES_VERSION=15.3-alpine
 REDIS_URL=redis://zammad-redis:6379
 REDIS_VERSION=7.0.5-alpine
 RESTART=always
-VERSION=6.1.0-58
+VERSION=6.2
```

i would go with the full version here too.
floating tags can lead to unwanted behaviour.

```suggestion
VERSION=6.2.0
```

---

We follow your advice here, @monotek. Can you please review again?